apps/studio/src/utils/generateAssetUrl.ts

```diff
@@ -1,7 +1,11 @@
 import { env } from "~/env.mjs"
 
+export const generateAssetBaseUrl = (): string => {
```

Comment:
question(non-blocking): curious why use a function instead of a constant?

Reply:
oops, was trying out different way to fix this weird storybook issue and forgot to update. thanks for spotting, updated!